docs/runbooks/set-environment-variable.md

```diff
@@ -31,29 +32,22 @@ $ task lagoon:cli:config
 # 2. Refresh your Lagoon token.
 $ lagoon login
 
-# 3. Then get the project id by listing your projects
-$ lagoon list projects
-
-# 4. Optionally, if you wish to set a variable for a single environment - eg a
-# pull-request site, list the environments in the project
-$ lagoon list environments -p <project name>
+# 3a. For project-level variables, use the following command, which creates
+#     the variable if it does not yet exist, or updates it otherwise:
+$ task lagoon:ensure:environment-variable \
+  PROJECT_NAME=<project name> \
+  VARIABLE_SCOPE=RUNTIME \
+  VARIABLE_NAME=<your variable name> \
+  VARIABLE_VALUE=<your variable value>
```

Comment:
Would be nice with an example of a runtime/environment level variable that you could set

Reply:
Hmm... you can kind of set anything you want, it is just passed as an environment variable.

I'm not sure if we have any good examples where we modify behavior based on env vars -- but it would be an obvious way to differentiate plans, if we need to, for example.